main.tf

```diff
@@ -46,7 +46,7 @@ resource "aws_vpc" "ttomczyk_vpc" {
 
 resource "aws_internet_gateway" "ttomczyk_aws_igw" {
   vpc_id = "aws_vpc.ttomczyk_vpc.id"
```

Comment:
Powinno być
`vpc_id = aws_vpc.ttomczyk_vpc.id`

tak jest praktycznie w całym pliczku, więc jak masz jakieś odwołania do innych resourców to bez ""